tests/spark/test_spark_connect_model_export.py

```diff
@@ -0,0 +1,145 @@
+import json
+import os
+
+import numpy as np
+import pandas as pd
+import pyspark
+import pytest
+from packaging.version import Version
+from pyspark.sql import SparkSession
+from pyspark.sql import functions as spark_f
```

Comment:
```suggestion
from pyspark.sql import functions as F
```

Is there any convention for an alias for `pyspark.sql.functions`?

Reply:
I am not sure what's the convention, but F is good to me